src/Command/Issue/IssueShowCommand.php

```diff
@@ -83,10 +88,37 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 'Title: '.$issue['title'],
                 'Link: '.$issue['url'],
                 '',
-                $issue['body'],
+                wordwrap($issue['body'], 100),
             ]
         );
 
+        if (true === $input->getOption('with-comments') && count($comments) > 0) {
```

Comment:
`$comments` is always defined when the option `with-comments` is true. I understand that you want a cleaner approach here :smile: